src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
There should be no imitation here for several reasons.

1. There is no Is-A correspondence between these classes, i.e. Lottery is not a Color Supplier. 
2. If we needed to logically inherit from a class that really was is-a, we couldn't, since we can only inherit from one class.
3. In this case, inheritance does not give us any advantages. We don't inherit any methods/override them and fields.